src/internal.c

```diff
@@ -14421,8 +14421,9 @@ int ProcessPeerCerts(WOLFSSL* ssl, byte* input, word32* inOutIdx,
                     }
                 }
                 else if (ret == ASN_PARSE_E || ret == BUFFER_E ||
-                         ret == MEMORY_E) {
-                    WOLFSSL_MSG("Got Peer cert ASN PARSE_E, BUFFER E, MEMORY_E");
+                         ret == MEMORY_E || ret == BAD_FUNC_ARG) {
+                    WOLFSSL_MSG("Got Peer cert ASN_PARSE_E, BUFFER_E, MEMORY_E,"
+                                " BAD_FUNC_ARG");
```

Comment:
Was there a configure option failing with BAD_FUNC_ARG once quic was added to enable-all. (tried with --enable-all --enable-quic && make check and did not see it). The change alters a lot of behavior such as ssl->peerVerifyRet value, verify callback calls, and potentially sending fatal cert alert.

Reply:
It wasn't failing `make check`. It was a clang sanitizer error in the multi-test. If `ProcessPeerCertParse()` returned `BAD_FUNC_ARG`, the code would enter the `XMEMCPY` on line 14494 with NULL parameters. I can move the `BAD_FUNC_ARG` check below the verify callback calls. I could also remove this change all together (since the error showed up when `--enable-haproxy` was part of this PR, and it no longer is) but we should definitely be checking for `BAD_FUNC_ARG` here

Reply:
Looking at the BAD_FUNC_ARG cas from ProcessPeerCertParse it looks like the case where that would be encountered is if args->dCert is NULL. In that case I think this is the right change in that the verify callbacks would not be used since there is no dCert set.